nordicNRPlus/Thingexists.spec.ts

```diff
@@ -0,0 +1,63 @@
+import type { DescribeThingCommand, IoTClient } from '@aws-sdk/client-iot'
+import assert from 'node:assert/strict'
+import { describe, it, mock } from 'node:test'
+import { thingExists } from './thingExists.ts'
+
+void describe('thingExists', () => {
+	void it('should check if thing exists in Iot Core', async () => {
+		const icSend = mock.fn(async () =>
+			Promise.resolve({
+				$metadata: {
+					httpStatusCode: 200,
+					requestId: '',
+					extendedRequestId: undefined,
+					cfId: undefined,
+					attempts: 1,
+					totalRetryDelay: 0,
+				},
+				attributes: { name: 'myDevice123' },
+				defaultClientId: '',
+				thingArn: '',
+				thingId: '',
+				thingName: '',
+				thingTypeName: '',
+				version: 3,
+			}),
```

Comment:
Simplify this to only provide the necessary data:
```suggestion
			Promise.resolve({
				attributes: { name: 'myDevice123' },
			}),
```